lib/index.js

```diff
@@ -73,7 +72,14 @@ exports.publish = function publish(basePath, config, done) {
   };
 
   // override defaults with any task options
-  var options = _.extend({}, defaults, config);
+  // TODO: Require Node >= 4 and use Object.assign
+  var options = {};
+  for (var d in defaults) {
+    options[d] = defaults[d];
+  }
+  for (var c in config) {
+    options[c] = config[c];
+  }
 
```

Comment:
One thing to keep in mind is [lodash.assign](https://www.npmjs.com/package/lodash.assign) and [Object.assign](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign) iterate over own properties of sources and not inherited properties.